spec/prawn/pdfa_1b_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'spec_helper'
+require 'prawn/vera_pdf'
+
+include Prawn::VeraPdf
+
+if vera_pdf_available?
```

Comment:
It's nice to let developers know whats's wrong. But please make sure CI has all tools installed to actually run the specs.